src/Symfony/Component/Messenger/Bridge/AmazonSqs/Transport/AmazonSqsFifoStamp.php

```diff
@@ -25,11 +25,13 @@ public function __construct(string $messageGroupId, string $messageDeduplication
         $this->messageDeduplicationId = $messageDeduplicationId;
     }
 
-    public function getAttributes(): array
+    public function getMessageGroupId(): string
     {
-        return [
-            'messageGroupId' => $this->messageGroupId,
-            'messageDeduplicationId' => $this->messageDeduplicationId,
-        ];
+        return $this->messageGroupId;
+    }
+
+    public function getMessageDeduplicationId(): string
```

Comment:
Is it possible to define a groupId but ignoring deduplicate I'd? Just asking.

if yes, this should be nullable

Reply:
Yes than default value will be used. Both parameters should be nullable i will update my PR.